charts/akeyless-secure-remote-access/Chart.yaml

```diff
@@ -14,14 +14,14 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 
-version: 0.28.5
+version: 0.29.0
 
 
 appVersion: v1.0.0_0.16.1_1.4.2
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
 annotations:
-  ztbVersion: v1.0.0
-  sshVersion: 0.16.1
+  ztbVersion: v2.0.0
+  sshVersion: v2.0.0
   ztpVersion: 1.4.2
```

Comment:
@OriMankali @renanaAkeyless @niv-ez please confirm the app verstion.
What the new version should be?

Reply:
I believe that ZTB and SSH can be the same version but ZTP must stay independent, If a change you make on ZTB will also be applied to SSH then it's OK but as long as their releases are not match then leave it as is for now

Reply:
I agree, you can also remove the 'v' prefix, it was an historic misalignment (just check there are no functional implications - e.g. validation of version, etc.)
We can just use ztbVersion and remove the ssh one

Reply:
Please make sure not to break bc, the version should be propagated to the same location the application was expecting

Reply:
@omriezra Please check the caller git repositories to make sure they will bump the same version and verify that the Helm chart release scrip in this repo knows how to handle it

Reply:
@OriMankali there are multiple places where we expect the version to start with "v" like here
https://github.com/akeylesslabs/zero-trust-bastion/blob/c4cf5ad327501545148c6451a5f76c568c57a252/.github/scripts/release_latest_version.sh#L15

@omriezra can we modify the pipline to not depend on this requirment?

Reply:
currently i leave it with both have the same version `v2.0.0`, @omriezra  please update me if we can remove the 'v' prefix.